features/checkout/seeing_order_summary/seeing_current_prices_of_products_after_catalog_promotion_becomes_ineligible.feature

```diff
@@ -0,0 +1,23 @@
+@checkout
+Feature: Seeing current prices of products after catalog promotion becomes ineligible
+    In order to buy products in its correct prices
+    As a Customer
+    I want to have products with its current prices in the cart
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "T-Shirt"
+        And this product has "PHP T-Shirt" variant priced at "$20.00" in "United States" channel
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And there is a catalog promotion "Winter sale" available in "United States" channel that reduces price by "25%" and applies on "PHP T-Shirt" variant
+        And I am a logged in customer
+
+    @ui @api
+    Scenario: Processing order with valid prices after catalog promotion becomes ineligibly
+        Given I have "PHP T-Shirt" variant of this product in the cart
+        When the administrator makes this catalog promotion unavailable in the "United States" channel
+        And I specified the billing address as "Ankh Morpork", "Frost Alley", "90210", "United States" for "Jon Snow"
+        And I proceed through checkout process
+        Then I should be on the checkout summary step
+        And I should see product "T-Shirt" with unit price "$20.00"
```

Comment:
Cannot we check it before the final stap? Even on the cart itself?

Reply:
TBH yes we can :D, we need to run `order Processor` after admin changed something so each step after addressing is valid, going through the entire checkout was easy in behat and also it checks what I want :)

Reply:
Maybe we could just update the cart? It's not a blocker for merge, but with all these steps it seems like it's necessary to go through the checkout to see the result. And there is `in the cart` in the scenario's description 💃